src/Discovery/Discovery.tsx

```diff
@@ -316,13 +316,51 @@ const Discovery: React.FunctionComponent<DiscoveryBetaProps> = (props: Discovery
     setSearchFilteredResources(results);
   };
 
-  const handleAccessLevelChange = (ev) => {
-    const value = ev.target.value as AccessLevel;
-    setAccessLevel(value);
+  const handleExportToWorkspaceClick = async () => {
```

Comment:
I assume this may take a while if the manifest is large? Do we have something in design to inform user that the operation is pending? like a disabled button or button with spinner?

Reply:
I can put a spinner on the button! I don't think this operation will take too long but it's a good thing to do anyway.